Playground: added new screen placeholders for 6 executions subscreens

joetaylor-harness opened the pull request.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/harness/canary?pullRequest=227) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/harness/canary?pullRequest=227) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/harness/canary?pullRequest=227) it.</sub>

joetaylor-harness commented:
- Built standalone `Executions` with the new `SandboxLayout`
- Updated routes to allow linking to from execution nav tabs to the placeholder views
- Added the executions list view as the contents of the summary tab, per figmas

<img width="1920" alt="image" src="https://github.com/user-attachments/assets/dbb21b83-4563-404c-85d1-52c6529508c4">
